projects/storefrontapp-e2e-cypress/cypress/integration/regression/product_configurator/product-configurator-vc.e2e-2205-spec-flaky.ts

```diff
@@ -91,13 +91,15 @@ context('Product Configuration - 2205', () => {
       cy.wait('@updateConfig');
```

Comment:
[product-configurator-vc.e2e-2205-spec-flaky.ts] is failing for me with a current mock backend
![Screenshot 2023-01-04 at 15 54 01](https://user-images.githubusercontent.com/33626130/210583079-ddd069c4-a7c3-4f82-a14f-7cfeef09b5e6.png)

Reply:
Tried to stabilize this test. Done!